torch/_higher_order_ops/auto_functionalize.py

```diff
@@ -34,7 +34,7 @@
 
 
 class AutoFunctionalized(HigherOrderOperator):
-    """auto_functionalized(op, mutated_args_names, kwargs)
+    """auto_functionalized(_nonfunctional_op, mutated_args_names, kwargs)
```

Comment:
_mutated_args_names

Reply:
And `**kwargs`

Reply:
Can you leave a comment somewhere saying the reason why these are underscore? (to prevent name collisions with **kwargs)